internal/rbd/rbd_util.go

```diff
@@ -175,6 +182,15 @@ type imageFeature struct {
 	dependsOn []string
 }
 
+// migrationHandle is a struct which consists of required fields of a rbd volume
+// from migrated volume handle.
+type migrationHandle struct {
+	Version   string
+	Mons      string
+	ImageName string
+	Pool      string
```

Comment:
is it pool Name or Pool ID?

Reply:
PoolName, renamed the field for more clarity